src/DatePicker/DatePicker.js

```diff
@@ -134,6 +134,14 @@ class DatePicker extends Component {
      */
     textFieldStyle: PropTypes.object,
     /**
+     * This object should contain methods needed to build the calendar system.
+     *
+     * Useful for building a custom calendar system. Refer to the
+     * [source code](https://github.com/callemall/material-ui/blob/master/src/DatePicker/dateUtils.js)
+     * for more information.
```

Comment:
Except that this file doesn't have more information (about custom calendars at least) 😄 

Might be worth linking to the source of your persian-datepicker-utils as an example?